ihmc-footstep-planning/src/main/resources/us/ihmc/footstepPlanning/SnappingTerrainExtractor.cu

```diff
@@ -9,17 +9,18 @@ extern "C"
 #define HEIGHT_MAP_WIDTH_IM_METERS 3
 #define FOOT_LENGTH 4
 #define FOOT_WIDTH 5
-#define MIN_DISTANCE_FROM_CLIFF_TOPS 6
-#define MIN_DISTANCE_FROM_CLIFF_BOTTOMS 7
-#define CLIFF_START_HEIGHT_TO_AVOID 8
-#define CLIFF_END_HEIGHT_TO_AVOID 9
-#define MIN_SUPPORT_AREA_FRACTION 10
-#define MIN_SNAP_HEIGHT_THRESHOLD 11
-#define SNAP_HEIGHT_THRESHOLD_AT_SEARCH_EDGE 12
-#define INEQUALITY_ACTIVATION_SLOPE 13
-#define STEPPING_COSINE_THRESHOLD 14
-#define STEPPING_CONTACT_THRESHOLD 15
-#define CONTACT_WINDOW_SIZE 16
+#define MIN_DISTANCE_FROM_BASE_OF_CLIFF 6
+#define MIN_DISTANCE_FROM_EDGE_OF_CLIFF 7
+#define SCALED_FOOT_POLYGON_PERCENTAGE 8
```

Comment:
Should be "SCALED_FOOT_POLYGON_DISTANCE" instead of "SCALED_FOOT_POLYGON_PERCENTAGE"?